lib/fizzy/types.hpp

```diff
@@ -154,6 +154,29 @@ struct Global
     } init;
 };
 
+enum class ImportType : uint8_t
```

Comment:
Maybe this should better be merged with `ExportType` (they are identical) into something like `ExternalType` (or `ExternalKind` to disambiguate from types of type section)